### ISSUE TITLE
Google Play Console Notice: Direct links don't work

### ISSUE THREAD

**Issue by ctechdev** [readium/kotlin-toolkit#426]
### What happened?

In our production app we received warnings regarding incorrect handling of direct links, domain (host), and pathPatterns within intent-filter.
  We are notified that:
- Domain failed validation
- Direct links don't work
Users won't be taken directly to your app from links associated with this path, but they will see the app switcher or be taken to the web browser. Once you fix these issues and release a new version, users will need to update the app for the links to work."
And the advice is this:
"<data> tag failed Add "/" to the beginning of the android:path attribute in the <data> tag. The attribute can also be android:pathPrefix or android:pathPattern."
We have replicated the same patterns as the testApp and so far we have not had any problems, how should we modify them to be valid?

currently it's like this:

`<data android:host="*" />`
`<data android:pathPattern=".*\\.EPUB" />`

Thanks in advance.

**Comment by uongsyphuong** [comment]
Has anyone solved this issue?

**Comment by stevenzeck** [comment]
I'm not sure this is a toolkit problem. What are you trying to do exactly? I imagine domain validation fails because of the wildcard `*` in `android:host`.

**Comment by mickael-menu** [comment]
Yes, it's not directly in the toolkit but in the test app. But I think it should still be mentioned in the documentation for anyone looking to implement side loading. I encountered the same Google Play warnings but couldn't find a solution. In practice though, it didn't cause any issues in the apps.
